.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
+auto.sync=false
+build.scans.enabled=false
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.project.dir=
+eclipse.preferences.version=1
+gradle.user.home=
+java.home=C\:/Program Files/AdoptOpenJDK/jdk-14.0.2.12-hotspot
```

Comment:
This is an machine specific setting. Is there anyway to use JAVA_HOME environment variable?

Reply:
Oh, didn't see that. Good catch! Now that I'm looking at this, all of these parameters seem to be dependent on the local setup, so maybe that file could be ignored instead. On Windows it is a very weird Java setup, so not sure about JAVA_HOME.

Reply:
I don't have JAVA_HOME set as an environment variable, but rather as some config in VS Code itself.